sdk/resourcemanager/Azure.ResourceManager.Core/src/Generated/Models/LocationExpanded.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace Azure.ResourceManager.Core
+{
+    /// <summary>
+    /// Represents an Azure geography region where supported resource providers live.
+    /// </summary>
+    public partial class LocationExpanded : Location
```

Comment:
give the autogenerator doesn't seal them, I am not sure we should do that here since its intended to be auto-gen at a later point being under the Generated folder